src/scalar.rs

```diff
@@ -37,12 +37,12 @@ impl Scalar {
     pub const MAX: Scalar = Scalar(MAX_RAW);
 
     /// Generates a random scalar
-    #[cfg(any(test, feature = "rand-std"))]
+    #[cfg(feature = "rand-std")]
     #[cfg_attr(docsrs, doc(cfg(feature = "rand-std")))]
     pub fn random() -> Self { Self::random_custom(rand::thread_rng()) }
 
     /// Generates a random scalar using supplied RNG
-    #[cfg(any(test, feature = "rand"))]
+    #[cfg(feature = "rand")]
     #[cfg_attr(docsrs, doc(cfg(feature = "rand")))]
     pub fn random_custom<R: rand::Rng>(mut rng: R) -> Self {
```

Comment:
Oh, I should've implemented `Distribution<Scalar> for Standard`. :man_facepalming:

Reply:
ITYM for `Uniform`, but honestly this seems like more complexity (and reading `rand` docs) (and hoping they never change) than it's worth.

Reply:
I think for both `Standard` and `Uniform`. If `rand` is still too far away from stabilizing maybe we should just put it off until then.